src/Core/DevOps/StaticAnalyze/PHPStan/core-rules.neon

```diff
@@ -18,3 +18,4 @@ rules:
     - Shopware\Core\DevOps\StaticAnalyze\PHPStan\Rules\AclValidPermissionsInRouteAttributesRule
     - Shopware\Core\DevOps\StaticAnalyze\PHPStan\Rules\ShopwareNamespaceStyleRule
     - Shopware\Core\DevOps\StaticAnalyze\PHPStan\Rules\AttributeFinalRule
+    - Shopware\Core\DevOps\StaticAnalyze\PHPStan\Rules\PublicServiceDecoratorRule
```

Comment:
shouldn't we put it to the common rules, which are also used in plugins? I guess, they would run in the same error, if they do not apply to it